unitTests/ui/DynamicFacetManagerTest.ts

```diff
@@ -93,25 +95,21 @@ export function DynamicFacetManagerTest() {
       expect(test.cmp.disabled).toBe(false);
     });
 
-    it('should reorder the facets correctly in the DOM depending on the query results', () => {
+    it(`when a facet is disabled
+    should not be sent in the request`, () => {
+      facets[0].disable();
+      const queryBuilder = new QueryBuilder();
       triggerAfterComponentsInitialization();
-      triggerQuerySuccess(queryFacetsResponse());
-
-      expect(managerContainerChildren()[0]).toBe(facets[1].element);
-      expect(managerContainerChildren()[1]).toBe(facets[2].element);
-      expect(managerContainerChildren()[2]).toBe(facets[0].element);
-    });
-
-    it('should remove the facets not in the response', () => {
-      triggerAfterComponentsInitialization();
-      triggerQuerySuccess([]);
+      $$(test.env.root).trigger(QueryEvents.doneBuildingQuery, {
+        queryBuilder
+      });
 
-      expect(managerContainerChildren().length).toBe(0);
+      const facetIsInRequest = !!findWhere(queryBuilder.facetRequests, { facetId: facets[0].options.id });
+      expect(facetIsInRequest).toBe(false);
     });
 
-    it('should retrieve the previously removed facets', () => {
+    it('should reorder the facets correctly in the DOM depending on the query results', () => {
```

Comment:
This test description should be more precise. What does "correctly" mean in this context? What are we "depending on" in the results?